<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qsoedit</class>
 <widget class="QDialog" name="Qsoedit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>430</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Редактирование QSO</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>380</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>380</height>
      </size>
     </property>
     <property name="title">
      <string>Параметры QSO</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="8" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>QTH локатор:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="qthloc_lineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>RDA/CNTY:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="rda_lineEdit"/>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Позывной станции:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="StationComboBox">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Позывной оператора:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="OperatorComboBox">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>Страна:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="countrylineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Континент:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="countryCodelineEdit">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Дата:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEdit">
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="maximumSize">
           <size>
            <width>65</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Время нач.:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTimeEdit" name="qso_timeStartEdit">
          <property name="displayFormat">
           <string>H:mm:ss</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Время окон.:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTimeEdit" name="QSO_timeEndEdit">
          <property name="displayFormat">
           <string>H:mm:ss</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Имя:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="name_lineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>QTH:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="qth_lineEdit"/>
        </item>
       </layout>
      </item>
      <item row="11" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="10" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Комментарий:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="comment_lineEdit"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Позывной:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="CalsignlineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="QSOSUUserIcon">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="resources.qrc">:/resources/images/loguser.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="QSOSUUserLabel">
          <property name="text">
           <string>Пользователь лога</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="SRRUserIcon">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="resources.qrc">:/resources/images/srr_user.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="SRRUserLabel">
          <property name="text">
           <string>Член СРР</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="7" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>RST прин.:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="rstr_lineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>RST отпр.:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="rsts_lineEdit"/>
        </item>
       </layout>
      </item>
      <item row="9" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>ITUZ:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ituzlineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>CQZ:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="cqzlineEdit"/>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Диапазон: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="bandCombo">
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="maxVisibleItems">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Частота: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="feqLine">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>Модуляция: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="modeCombo">
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="maxVisibleItems">
           <number>20</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="QRZUpdateButton">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Обновить с Callbook</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>200</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QGraphicsView" name="graphicsView">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>StationComboBox</tabstop>
  <tabstop>OperatorComboBox</tabstop>
  <tabstop>CalsignlineEdit</tabstop>
  <tabstop>countrylineEdit</tabstop>
  <tabstop>countryCodelineEdit</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>qso_timeStartEdit</tabstop>
  <tabstop>QSO_timeEndEdit</tabstop>
  <tabstop>name_lineEdit</tabstop>
  <tabstop>qth_lineEdit</tabstop>
  <tabstop>rstr_lineEdit</tabstop>
  <tabstop>rsts_lineEdit</tabstop>
  <tabstop>qthloc_lineEdit</tabstop>
  <tabstop>rda_lineEdit</tabstop>
  <tabstop>ituzlineEdit</tabstop>
  <tabstop>cqzlineEdit</tabstop>
  <tabstop>comment_lineEdit</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>QRZUpdateButton</tabstop>
  <tabstop>graphicsView</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
